INFO  | File run_exp_CCN_Project.py | Line 15    | Writing log file to ./exp/GTS_test_1223_125605/log_exp_1010.txt
INFO  | File run_exp_CCN_Project.py | Line 16    | Exp instance id = GTS_test_1223_125605
ERROR | File run_exp_CCN_Project.py | Line 25    | Traceback (most recent call last):
  File "/Users/juhyeonkim/Dropbox/내 Mac (JuHyeonui-iMac.local)/Documents/GraphStructureLearning/run_exp_CCN_Project.py", line 19, in main
    runner = GTS_Runner(config=config)
  File "/Users/juhyeonkim/Dropbox/내 Mac (JuHyeonui-iMac.local)/Documents/GraphStructureLearning/runner/CCN_Project_runner.py", line 53, in __init__
    train_data, valid_data, test_data = self.split_train_valid_test(dataset)
  File "/Users/juhyeonkim/Dropbox/내 Mac (JuHyeonui-iMac.local)/Documents/GraphStructureLearning/runner/CCN_Project_runner.py", line 71, in split_train_valid_test
    if self.dataset_conf.pred == 'vel':
AttributeError: 'EasyDict' object has no attribute 'pred'

